plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -189,6 +190,10 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             Hash.sha3
             (runCostingFunOneArgument . paramSHA3)
+    toBuiltinMeaning Blake2b_256 =
+        makeBuiltinMeaning
+            Hash.blake2b
+            (runCostingFunOneArgument . paramBlake2b)
```

Comment:
you could just stick `mempty` in here (aka "no model"), not sure what you'd prefer @kwxm ?

Reply:
I can have a go at adding a cost model.  There are already some builtins that have no cost model and I've been meaning to fix that (and there are even more with built-in `Data` now).

Reply:
Yeah, but then the question is: is it better in PRs like this to just put `mempty` or to put in some of the scaffolding as Nikos has done? What's easier for you?

Reply:
I will change this to `mempty` and add a TODO at this loc . I will keep the rest boilerplate